Examples/Python/src/TrackFinding.cpp

```diff
@@ -315,6 +389,8 @@ void addTrackFinding(Context& ctx) {
   ACTS_PYTHON_DECLARE_ALGORITHM(ActsExamples::SeedsToPrototracks, mex,
                                 "SeedsToPrototracks", inputSeeds,
                                 outputProtoTracks);
+
+
```

Comment:
please remove unrelated formatting changes unless they significantly improve the code style.